cluster-autoscaler/core/scaledown/actuation/drain_test.go

```diff
@@ -18,6 +18,7 @@ package actuation
 
 import (
 	"fmt"
+	"math"
```

Comment:
I'd add a test case for how DS pods are handled if priorityEvictorEnabled is set to true - that we are treating them the same as non-DS pods.

Reply:
Done